__tests__/lib/messageUtils.test.ts

```diff
@@ -129,9 +129,43 @@ describe('getDashboard', () => {
     const startDate = "2024-03-08"
     const dashboardId = getDashboardIdForTemplate(template)
 
-    const expectedLink = `https://mozilla.cloud.looker.com/dashboards/${dashboardId}?Submission+Timestamp+Date=2024-03-08+to+today&Message+ID=%25${encodeURIComponent(msgId)}%25&Normalized+Channel=&Experiment=&Branch=`
+    const expectedLink = `https://mozilla.cloud.looker.com/dashboards/${dashboardId}?Submission+Timestamp+Date=2024-03-08%20to%20today&Message+ID=%25${encodeURIComponent(msgId)}%25&Normalized+Channel=&Experiment=&Branch=`
 
     const result = getDashboard(template, msgId, undefined, undefined, undefined, startDate, null)
     expect(result).toEqual(expectedLink)
   })
 })
+
+describe("getSubmissionTimestampDateFilter", () => {
+  it("returns the appropriate date filter when startDate and endDate are null", () => {
```

Comment:
```suggestion
  it("returns the default date filter when startDate and endDate are null", () => {
```

Reply:
It would be helpful to be explicit about what behavior is expected for each of the rest of the tests so that if one of the tests breaks, it's immediately clear what's wrong.